Breakout/Packages/manifest.json

```diff
@@ -7,6 +7,7 @@
     "com.unity.collab-proxy": "1.2.16",
     "com.unity.ide.rider": "1.1.4",
     "com.unity.ide.vscode": "1.1.4",
+    "com.unity.ml-agents": "file:/Users/fate/ml-agents/com.unity.ml-agents",
```

Comment:
ML-Agents is a local package, as you can see in my manifest. Not sure how to convert it to remote (if it's possible); can you check if there are any issues?